{"uuid":"bf338a80-a808-4aa1-b1c3-9eaa3bc589e5","historyId":"cf235ef5167cd0b6094cbc03f6199248","testCaseId":"[engine:junit-jupiter]/[class:com.example.data.repository.RevisionRepositoryTest]/[method:insert should throw exception when dao fails()]","testCaseName":"insert should throw exception when dao fails()","fullName":"com.example.data.repository.RevisionRepositoryTest.insert should throw exception when dao fails","labels":[{"name":"junit.platform.uniqueid","value":"[engine:junit-jupiter]/[class:com.example.data.repository.RevisionRepositoryTest]/[method:insert should throw exception when dao fails()]"},{"name":"host","value":"Maksims-MacBook-Air.local"},{"name":"thread","value":"2831@Maksims-MacBook-Air.local.Test worker(1)"},{"name":"framework","value":"junit-platform"},{"name":"language","value":"java"},{"name":"package","value":"com.example.data.repository.RevisionRepositoryTest"},{"name":"testClass","value":"com.example.data.repository.RevisionRepositoryTest"},{"name":"testMethod","value":"insert should throw exception when dao fails"},{"name":"suite","value":"com.example.data.repository.RevisionRepositoryTest"}],"links":[],"name":"insert should throw exception when dao fails()","status":"passed","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1737056717232,"stop":1737056717251}